mm/mm_heap/mm.h

```diff
@@ -129,10 +129,10 @@
  */
 
 #ifdef CONFIG_MM_SMALL
-# define MM_ALLOC_BIT    0x8000
+# define MM_ALLOC_BIT    0x0001
```

Comment:
Please move `MM_ALLOC_BIT` out of `CONFIG_MM_SMALL` ifdef and `#define MM_ALLOC_BIT 0x1` or `1 << 0`

Reply:
Done